.github/workflows/node.js.yml

```diff
@@ -13,14 +13,23 @@ jobs:
         node-version: [18.x]
 
     services:
-      postgres:
+      postgres-integration:
         image: postgres:10.8
         env:
-          POSTGRES_USER: coronaschool
-          POSTGRES_PASSWORD: coronanervt
-          POSTGRES_DB: coronaschool-test
+          POSTGRES_USER: test
+          POSTGRES_PASSWORD: test
+          POSTGRES_DB: integration
         ports:
-          - 5432:5432
+          - 5001:5432
+        options: --health-cmd pg_isready --health-interval 10s --health-timeout 5s --health-retries 5
+      postgres-verify-schema:
```

Comment:
I couldn't find a good way to start a single Postgres docker-container with multiple DBs, and using the same DB for different things could be error prone ... So two DBs it is ...